src/services/reporting/carSales.ts

```diff
@@ -39,10 +39,7 @@ const envelopeRequest = async (handler): Promise<WithValidationError<null>> => {
       result: null,
     }
   } catch (error) {
-    return handleValidationError({
-      error,
-      options: { swallowErrors: true },
-    })
+    return handleValidationError(error)
```

Comment:
Are you sure that it is related to the `swallowErrors` option? How it is related?

Reply:
When `swallowErrors` is true then the response is undefined. 
But to be honest I don't understand why. Here : https://github.com/carforyou/carforyou-api-client-pkg/blob/2de06f89567613bb5036ccf8090b0d01c9154908/src/base.ts#L207, our `error.name` is "ResponseError" so it should not matter